pyhive/hive.py

```diff
@@ -207,6 +196,109 @@ def sasl_factory():
             self._transport.close()
             raise
 
+    @staticmethod
+    def create_http_transport(host, port, http_path, username, password,
+                              kerberos_service_name, auth):
+        if port is None:
+            port = 10001
+        if auth is None:
+            auth = 'NONE'
+        if http_path is None:
+            http_path = '/'
```

Comment:
It could be better to default this to `/cliservice`, as that seems to be the default path HiveServer2 provides